README.md

```diff
@@ -198,6 +206,26 @@ A numeric value that can be parsed by [`(float)` cast, equivalent to `floatval`]
 ###### `bool` default
 `true`, `on`, `yes` or `1` will result in true. Other values, as long as there is something, will result default false. (If there is nothing, the variable will not be optional)
 
+###### `timestamp` default
+A numeric value that can be parsed by [`(int)` cast, equivalent to `intval`](https://php.net/intval) or [`(float)` cast, equivalent to `floatval`](https://php.net/floatval)
+
+**Allowed default values:**
+- `0`
+- `NOW`
+
+###### `list`
+An array of values whose type must be specified in the same way as the variable declaration.
```

Comment:
Where is this allowed? Just reading the documentation, I would expect this to be usable in `INSERT INTO tbl (a, b) VALUES (:a, :b)` where `a` and `b` are lists. But in fact this is only allowed in the `IN` context.
Furthermore, should it be used as `WHERE col IN :list` or `WHERE col IN (:list)`? Please clarify.